### ISSUE TITLE
FR: How to disable/override keyboard navigation on `igc-dockmanager`?

### ISSUE THREAD

**Issue by luiscla27** [IgniteUI/igniteui-webcomponents#1317]
Hi!

This a Feature Request to make the[ KEYBOARD NAVIGATION of DockManager](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/layouts/dock-manager#docking) configurable. Ideally, the shortcuts used should be configurable by the developer implementing the component, and even disable them! like instead of using `ctrl + shift + arrow left`, maybe that shortcut is already occupied, and I want to use another one.

I'm using the Angular implementation of Dock Manager, also read the web-component docs. But, I couldn't find a way to turn off the keyboard navigation:
- https://www.infragistics.com/products/ignite-ui-angular/angular/components/dock-manager
- https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/layouts/dock-manager#docking

I want to disable all of it.

The reason is that I'm using it to put text boxes inside of it, and `ctrl + shift + arrow left/right` is also a shortcut used by certain users to move the cursor to words around the textbox. This issue had happened in other products, and usually occurs due to bugs, this are some examples:
- Same [issue on Notepad++ ](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4649) (fixed).
- Same [issue on VSCode](https://github.com/microsoft/vscode/issues/41199) (fixed).

Please let me know if I should move this [the Angular repo](https://github.com/IgniteUI/igniteui-angular/issues).

- [ ] Research
- [x] Specification
- [ ] Component and tests implementation
- [x] API Documentation
- [x] Design implementation
